app/uk/gov/hmrc/economiccrimelevyregistration/navigation/EntityTypePageNavigator.scala

```diff
@@ -65,7 +65,8 @@ class EntityTypePageNavigator @Inject() (
               .createPartnershipJourney(entityType, mode)
               .map(createJourneyResponse => Call(GET, createJourneyResponse.journeyStartUrl))
 
-          case Other => ???
+          case Other =>
```

Comment:
Add a corresponding unit test in `EntityTypePageNavigatorSpec`

Reply:
Added.